src/utils.ts

```diff
@@ -1,7 +1,7 @@
 import { Readable } from 'stream'
-import tar from 'tar-stream'
-import gunzip from 'gunzip-maybe'
-import AdmZip from 'adm-zip'
+import * as tar from 'tar-stream'
+import * as AdmZip from 'adm-zip'
+const gunzip = require('gunzip-maybe')
```

Comment:
Could we maybe use [Node's built-in zlib](https://nodejs.org/api/zlib.html) (`zlib.unzip()`/`zlib.gunzip()`) here?

Reply:
I think this was a bit out of scope... but done!

Reply:
Nice catch, btw!